hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -101,20 +94,15 @@ public final class OzoneConsts {
 
 
   public static final String CONTAINER_EXTENSION = ".container";
-  public static final String CONTAINER_META = ".meta";
-
-  // Refer to {@link ContainerReader} for container storage layout on disk.
-  public static final String CONTAINER_PREFIX  = "containers";
   public static final String CONTAINER_META_PATH = "metadata";
   public static final String CONTAINER_TEMPORARY_CHUNK_PREFIX = "tmp";
   public static final String CONTAINER_CHUNK_NAME_DELIMITER = ".";
-  public static final String CONTAINER_ROOT_PREFIX = "repository";
 
   public static final String FILE_HASH = "SHA-256";
   public static final String MD5_HASH = "MD5";
   public static final String CHUNK_OVERWRITE = "OverWriteRequested";
 
-  public static final int CHUNK_SIZE = 1 * 1024 * 1024; // 1 MB
+  public static final int CHUNK_SIZE = 1 * 1024 * 1024;
```

Comment:
Please include this as well.
```suggestion
  public static final int CHUNK_SIZE = 1 * 1024 * 1024; // 1 MB
```